index.html

```diff
@@ -55,9 +55,9 @@ <h1>Introduction</h1>
 
 			<p>This specification is a module of the <a>Web Publication</a> specification. It will describe the requirements to create an Audiobook profile of a Web Publication.</p>
 
-			<p>An Audiobook is defined as a collection of audio resources grouped together by a reading order, metadata, and resources, all contained in a manifest. This Audiobook can live on the Open Web Platform, or packaged using the <a href="https://w3c.github.io/wpub/index.htmlpackaged-web-publications">Packaged Web Publications</a> specification.</p>
+			<p>An Audiobook is defined as a collection of audio resources grouped together by a reading order, metadata, and resources, all contained in a manifest. This Audiobook can live on the Open Web Platform, or packaged using a specification to be determined by the Working Group.</p>
 
-			<p>This specification is intended to bring clear standards to the audiobooks distribution model on the web. It should facilitate different user agent architectures for the consumption of Audiobooks. The primary goal is to bring clarity to a part of the publishing industry currently underserved by standards, while opening Audiobooks to the Open Web Platform and new user agents.</p>
+			<p>This specification is intended to bring clear standards to the audiobooks distribution model on the web and between businesses. It should facilitate different user agent architectures for the consumption of Audiobooks. The primary goal is to bring clarity to a part of the publishing industry currently underserved by standards, while opening Audiobooks to the Open Web Platform and new user agents.</p>
```

Comment:
"clear standards" is one of those subjective kinds of claims that has a way of arousing passions. Just saying it "standardizes" might be all you need here.